witness/src/Dscp/Snowdrop/Storage/Avlp.hs

```diff
@@ -1,440 +0,0 @@
-{-# OPTIONS_GHC -fno-warn-orphans #-}
-{-# LANGUAGE AllowAmbiguousTypes #-}
-{-# LANGUAGE NamedFieldPuns      #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-
--- This module is a complete BARDAQ, please rewrite it.  And after you
--- do, add docs and sections. It's harder to read/understand than you
--- think!
-
--- | AVL plus integration into snowdrop.
-
-module Dscp.Snowdrop.Storage.Avlp
```

Comment:
Akhem:
* Was this ever discussed before? 
* Did you already break everything or we still have time to bring this module back?

cc @Heimdell

Reply:
What? You had a terrible copy-pasta in this module. What do you want to discuss?

Reply:
At least we had control over it. It's up to Kirill to decide, I think.

Reply:
I disagree. "We control" is not an excuse for maintaining a number of code duplicates. You're always welcome to make a PR against snowdrop if some change is desirable.